### PR TITLE
Added Readme file issue #1885

### PR DESCRIPTION
**Pull request by kmrakash**
Fixes #

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [ ] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- 
- 

#### Files Added:
- README File for FLyod's Cycle Detection
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by SakshiShreya** [comment]
Please mention the issue number in PR template. And complete it properly.

File: Floyd_Cycle_Detection_Algorithm/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+The name already give idea about the algorithm , It determines if a singly linked list is circular and if it is,  It is going to return the node where the cycle begins
+It was invented by **Robert W. Floyd** in late 1960s.
+The algorithm is also popularly known as **The Tortoise and the Hare (Floyd’s Algorithm)**
+
+## Complexity
+
+Time Complexity - O(N) , N :- Length of List <br/>
+Space Complexity - O(1)
+
+
```
Comment (SakshiShreya) [inline Floyd_Cycle_Detection_Algorithm/README.md]: Remove these extra new-lines.

**Comment by jainaman224** [comment]
@kmrakash Are you working on it?

**Comment by SakshiShreya** [comment]
@kmrakash since there has been no activity, I am closing this issue.

**Pull request closed without merging**